core/src/rpc/verifier.rs

```diff
@@ -216,7 +215,7 @@ impl ClementineVerifier for Verifier {
 
                 nonce_idx += 1;
                 tracing::debug!(
-                    "Verifier {} signed sighash {} of {}",
+                    "Verifier {} signed and sent sighash {} of {} through rpc deposit_sign",
```

Comment:
you can add a `#[tracing::instrument(skip_all, fields(verifier_idx = self.idx))]` to the fn header instead of writing `through rpc deposit_sign`